sample/F8/tracks

```diff
@@ -0,0 +1,209 @@
+[{
+		"id": 1,
+		"name": "ADS & MONETIZATION",
```

Comment:
This is weird. I don't say anything of that sorts on my screen. What we can do is that you could tell me the exact areas that it's giving the errors and I will re-write them?
<img width="597" alt="screen shot 2017-06-05 at 3 54 17 pm" src="https://cloud.githubusercontent.com/assets/21010060/26780392/478e3420-4a07-11e7-9b8e-dbea3cf563f0.png">